### PR TITLE
Gio/16-10-2023

### PR DESCRIPTION
**Pull request by sinkingsugar**
**Description**

**Test plan**

### CODE REVIEW

File: shards/core/runtime.cpp [hunk 1]
```diff
@@ -3032,7 +3033,6 @@ void shInit() {
   shInterface.set("createMesh", emscripten::val(reinterpret_cast<uintptr_t>(iface->createMesh)));
   shInterface.set("destroyMesh", emscripten::val(reinterpret_cast<uintptr_t>(iface->destroyMesh)));
   shInterface.set("schedule", emscripten::val(reinterpret_cast<uintptr_t>(iface->schedule)));
-  shInterface.set("unschedule", emscripten::val(reinterpret_cast<uintptr_t>(iface->unschedule)));
```
Comment (guusw) [inline shards/core/runtime.cpp:3035]: What's the new way to unscheduled now? Or just not possible in the new C API?

**Review by guusw — APPROVED** [review]
Looks good, just a question

File: shards/core/runtime.cpp (hunk 1 above)
Comment (sinkingsugar) [reply shards/core/runtime.cpp:3035]: it's better to stop the wire, always was

Comment (sinkingsugar) [reply shards/core/runtime.cpp:3035]: but I could revert that bit

**Pull request merged**